### PR TITLE
Boseong Yun - Homework 1

### PR DESCRIPTION
**Pull request by boseongyun**
The most challenging part of the assignment was to understand the connection between the R Studio Server and Github. Specifically, it was challenging how forking, committing, pulling, and pulling requests manifested on both of the platforms. Luckily, I was able to get tremendous help from the **video tutorial** professor Soltoff uploaded on the class materials. Additionally, I was able to get help from **teaching assistants via screen sharing feature on Zoom**. This task would have been impossible without the very accessible and friendly assistance since I had hard time translating the written explanations into readily viewable outputs.

The assignment itself, however, was relatively easy. In particular, I was able to experiment with all the features required for this assignment by referring to **Rmarkdown Cheatsheet** provided by the R studio  alone. I was able to learn a lot about how much leeway we had over in making fine adjustments to the ways we could present our works. Overall, I learned a lot about R Studio Server, Github, and Rmarkdown!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,31 @@
...
+  * Rmarkdown
+  * Final Thoughts
+  
+  
+## Introduction
+![me](Michigan.jpg)
+
+My name is **Boseong Yun** and please feel free to call me **Peter**! I am a first-year grdaute student studying public policy at the Harris School of Public Policy. I am studying to quantify gender inequalites in text, image, sound, and other kinds of unstructured data. Yes, I have a very long way to go. For more information about me, you can check out my [Youtube channel](https://www.youtube.com/channel/UCfv5YsyMCcQPNyT-g0cEYGA) which might be more boring than our new lives quarantined at home. I will try to be funny...! Also, please do not hesitate to talk to me if you have any questions that I can help you with and it is really nice to meet you all! 
+
+## Reflections
```
Comment (deblnia) [inline README.md:22]: For future assignments, you can leave the reflection in the pull request text box. Not a big deal at all, but it usually satisfying to just see a clean report.

**Review by deblnia — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     X      |              |            |
| **Coding strategy**         |    X       |              |            |
| **Presentation**            |         X  |              |            |
| **Achievement, creativity** |     X      |              |            |
| **Reproducibility**         |       X    |              |            |

Remarks:

* Great work -- you successfully used git and github and used a bunch of different elements of markdown syntax!  😄 
* Awesome use of images -- that can be tricky!

File: README.md (hunk 1 above)
Comment (boseongyun) [reply README.md:22]: Sure thing! I will leave the reflection part in the pull request text box. Thank you, deblnia!

Comment (jordivasquez) [inline README.md:22]: I really liked the use of headers and subheaders here!

File: README.md [hunk 2]
```diff
@@ -1 +1,31 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## Overview
+* Introduction
+  * Name: Boseong Yun (Peter)
+  * Major: Public Pollicy
+  * Acadmeic interests: gender equality
+  * hobby: Youtube
```
Comment (jordivasquez) [inline README.md:8]: For the list, maybe keep capitalization consistent (not a big deal).

File: README.md [hunk 3]
```diff
@@ -1 +1,31 @@
...
+  
+## Introduction
+![me](Michigan.jpg)
+
+My name is **Boseong Yun** and please feel free to call me **Peter**! I am a first-year grdaute student studying public policy at the Harris School of Public Policy. I am studying to quantify gender inequalites in text, image, sound, and other kinds of unstructured data. Yes, I have a very long way to go. For more information about me, you can check out my [Youtube channel](https://www.youtube.com/channel/UCfv5YsyMCcQPNyT-g0cEYGA) which might be more boring than our new lives quarantined at home. I will try to be funny...! Also, please do not hesitate to talk to me if you have any questions that I can help you with and it is really nice to meet you all! 
+
+## Reflections
+
+- **Github Workflow:**
+This homework would have been impossible had it not been for the very instructive **video tutorial** uploaded on the course website. Specifically, it was hard to understand the dynamics of github and its versatilty with the R enviornment in forking, committing, pulling, and performing other commands. The video tutorial provided clear mapping to what and why I was actually doing by using these commands.
```
Comment (jordivasquez) [inline README.md:25]: I agree the video was great!

**Review by jordivasquez — COMMENTED** [review]
I thought it was a great biography! It covered a lot of content but was well organized (I liked that you  started with a summarizing list, basically a tl;dr). Also its awesome that you have a youtube channel!

**Pull request closed without merging**